obsreport/obsreport_scraper.go

```diff
@@ -47,25 +66,67 @@ type ScraperSettings struct {
 
 // NewScraper creates a new Scraper.
 func NewScraper(cfg ScraperSettings) (*Scraper, error) {
-	return &Scraper{
+	return newScraper(cfg, featuregate.GetRegistry())
+}
+
+// Deprecated: [v0.65.0] use NewScraper.
+func MustNewScraper(cfg ScraperSettings) *Scraper {
```

Comment:
just clarifying this wasn't added here, the code moving in this PR makes it look like it was

Reply:
Unsure why its all highlighted green, something weird must have happened when rebasing. But my branch is up to date